packages/twenty-front/src/modules/object-record/record-table/record-table-row/components/RecordTableRowWrapper.tsx

```diff
@@ -82,6 +83,12 @@ export const RecordTableRowWrapper = ({
             trRef.current = node;
             elementRef(node);
             draggableProvided.innerRef(node);
+
+            if (typeof ref === 'function') {
```

Comment:
what are we trying to achieve here? looks hacky

Reply:
It was actually for motion, so when we pass our custom component to motion this one can actually mange it (based on the ref).
But as we already provide a function to the ref, we need to manually instantiate the forward ref.
This can be removed I guess when we find a proper way to animated the records.

Reply:
yes let's remove it in this case! this seems to hacky for me!